clc/modules/object-storage/src/main/java/com/eucalyptus/objectstorage/pipeline/binding/ObjectStorageOPTIONSBinding.java

```diff
@@ -74,14 +74,17 @@
   @Override
   protected Map<String, String> populateOperationMap() {
     Map<String, String> newMap = new HashMap<>();
+
+    // Cross-Origin Resource Sharing (CORS) pre-flight OPTIONS request,
+    // can be sent against either a bucket or object, same behavior.
+    newMap.put(BUCKET + HttpMethod.OPTIONS.toString(), "PreflightCheckCors");
+    newMap.put(OBJECT + HttpMethod.OPTIONS.toString(), "PreflightCheckCors");
+
```

Comment:
Supported and unsupported operations are a constant. Generate a copy of these maps statically when the class loads instead of creating a new one every single time